contrib/bundle/build

```diff
@@ -162,8 +160,8 @@ pushd $TAR_DIR
 export DESTDIR=test/
 ./install
 EXP_CNT=64
-if [[ $ARCH == "$ARCH_AMD64" ]]; then
-    EXP_CNT=65
+if ! command -v runc; then
```

Comment:
@saschagrunert is this still necessary? should we unconditionally install?

Reply:
I think we should install it in any case, yes.